packages/api/test/__fixtures__/sdk/petstore/index.ts

```diff
@@ -0,0 +1,592 @@
+   * within your API request.
+   *
+   * With the exception of OpenID and MutualTLS, it supports all forms of authentication
+   * supported by the OpenAPI specification.
+   *
+   * @example <caption>HTTP Basic auth</caption>
+   * sdk.auth('username', 'password');')
+   *
+   * @example <caption>Bearer tokens (HTTP or OAuth 2)</caption>
+   * sdk.auth('myBearerToken');
+   *
+   * @example <caption>API Keys</caption>
+   * sdk.auth('myApiKey');
+   *
+   * @see {@link https://spec.openapis.org/oas/v3.0.3#fixed-fields-22}
+   * @see {@link https://spec.openapis.org/oas/v3.1.0#fixed-fields-22}
+   * @param values Your auth credentials for the API; can specify up to two strings or numbers.
+   */
+  auth(...values: string[] | number[]) {
+    this.core.setAuth(...values);
+    return this;
+  }
+
+  /**
+   * If the API you're using offers alternate server URLs, and server variables, you can tell
+   * the SDK which one to use with this method. To use it you can supply either one of the
+   * server URLs that are contained within the OpenAPI definition (along with any server
+   * variables), or you can pass it a fully qualified URL to use (that may or may not exist
+   * within the OpenAPI definition).
+   *
+   * @example <caption>Server URL with server variables</caption>
+   * sdk.server('https://{region}.api.example.com/{basePath}', {
+   *   name: 'eu',
+   *   basePath: 'v14',
+   * });
+   *
+   * @example <caption>Fully qualified server URL</caption>
+   * sdk.server('https://eu.api.example.com/v14');
+   *
+   * @param url Server URL
+   * @param variables An object of variables to replace into the server URL.
+   */
+  server(url: string, variables = {}) {
+    this.core.setServer(url, variables);
+  }
+
+  /**
+   * Add a new pet to the store
+   *
+   * @param body Request body payload data.
+   */
+  post(path: string, body: Pet): Promise<unknown>;
+  /**
+   * Updates a pet in the store with form data
+   *
+   * @param body Request body payload data.
+   * @param metadata Object containing all path, query, header, and cookie parameters to supply.
+   */
+  post(path: string, body, metadata: UpdatePetWithFormMetadataParam): Promise<unknown>;
+  /**
+   * This can only be done by the logged in user.
+   *
+   * @summary Create user
+   * @param body Request body payload data.
+   */
+  post(path: string, body: User): Promise<unknown>;
+  /**
+   * Creates list of users with given input array
+   *
+   * @param body Request body payload data.
+   */
+  post(path: string, body: CreateUsersWithArrayInputBodyParam): Promise<unknown>;
+  /**
+   * Creates list of users with given input array
+   *
+   * @param body Request body payload data.
+   */
+  post(path: string, body: CreateUsersWithListInputBodyParam): Promise<unknown>;
+  /**
+   * Uploads an image
+   *
+   * @param body Request body payload data.
+   * @param metadata Object containing all path, query, header, and cookie parameters to supply.
```

Comment:
I really don't like these docblocks but I also don't know what else to do aside from maybe dumping the that the accompanying types (like `UploadFileBodyParam` have in here -- and I definitely know I won't like that.